src/pages/dashboard/forms/confirm-unstake-form.tsx

```diff
@@ -21,16 +20,7 @@ const ConfirmUnstakeForm = (props: Props) => {
   const handleAction = async () => {
     setError('');
 
-    const [err] = await go(onConfirm(amount));
-    if (err) {
-      if (isUserRejection(err)) {
-        notifications.info({ message: messages.TX_GENERIC_REJECTED });
-        return;
-      }
-      setError(messages.TX_GENERIC_ERROR);
```

Comment:
There is a small change in behaviour here. Previously, we showed an error in the form as helper text and haven't shown an error notification.

Now, there will be no helper text in modal when the `onConfirm` (a.k.a the transaction) fails. We will show the error notification instead. This makes it consistent with new proposal form as well and the rule is:

Show any validation errors in the modal, but onConfirm error handling is not part of the validation anymore and will trigger a toast message error.